packages/fluentui/react-northstar/src/components/Form/utils/formFieldBase.tsx

```diff
@@ -99,15 +104,17 @@ const _FormFieldBase = compose<'div', FormFieldBaseProps, {}, {}, {}>(
               ...slotProps.label,
             }),
         })}
-        {createShorthand(composeOptions.slots.control, control || {}, {
-          defaultProps: () =>
-            getA11yProps('control', {
-              error: !!errorMessage || null,
-              ref,
-              ...unhandledProps,
-              ...slotProps.control,
-            }),
-        })}
+        <FormFieldBaseProvider value={childProps}>
```

Comment:
Can we please add comment in the code that will explain why we use context for this property.

Reply:
Done